### ISSUE TITLE
Upgrading to Jetty 11.0.7: does not define or inherit an implementation of the resolved method 'abstract java.lang.String getRequestedApiVersion()'

### ISSUE THREAD

**Issue by catapop84** [eclipse/jetty.project#7004]
**Jetty version(s)**
11.0.7
**Java version/vendor** `(use: java -version)`
16.0.2
**OS type/version**
Windows 10
**Description**
Upgrading from jetty 11.0.6 to 11.0.7 throws this error: 

```
Unexpected problem occured during version sanity check
Reported exception:
java.lang.AbstractMethodError: Receiver class org.apache.logging.slf4j.SLF4JServiceProvider does not define or inherit an implementation of the resolved method 'abstract java.lang.String getRequestedApiVersion()' of interface org.slf4j.spi.SLF4JServiceProvider.
	at org.slf4j.LoggerFactory.versionSanityCheck(LoggerFactory.java:297)
	at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:141)
	at org.slf4j.LoggerFactory.getProvider(LoggerFactory.java:421)
	at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:407)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:356)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:382)
```

**How to reproduce?**
Upgrade to Jetty 11.0.7 , downgrading back to 11.0.6 the warning doesn't appear

**Comment by olamy** [comment]
how do you reproduce this?

**Comment by markslater** [comment]
I think this is caused by Jetty's SLF4J dependency changing to `2.0.0-alpha5` in `11.0.7` (previously it depended on `2.0.0-alpha1`).

I saw a very similar failure when upgrading due to declaring a dependency on `org.slf4j:slf4j-nop:2.0.0-alpha1` in my project.  It was fixed by replacing that with `org.slf4j:slf4j-nop:2.0.0-alpha5`.

If you have SLF4J dependencies declared in your project, you might fix the problem by upgrading them to latest.

**Comment by joakime** [comment]
When we upgraded to `slf4j-api` `2.0.0-alpha4` that typo was fixed by slf4j, so our implementation at `jetty-slf4j-impl` was updated as well.

See commit 518c13a636f06dcbed7abb505b5af93360d8c94c

Jetty releases 10.0.7 and 11.0.7 contain this fix.

Jetty 10.0.7 and 11.0.7 shipped with slf4j-api at `2.0.0-alpha5`

https://github.com/eclipse/jetty.project/blob/jetty-11.0.7/pom.xml#L118
